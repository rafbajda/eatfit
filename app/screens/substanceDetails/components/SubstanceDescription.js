import React from 'react';
import { Text } from 'native-base';
import {DescriptionContainer, SubstanceInformationLabel} from '../styles/substanceDetailsStyles';

const SubstanceDescription = props => {
    const { substanceDescription } = { ...props };
    return (
        <DescriptionContainer>
            <SubstanceInformationLabel>Description:</SubstanceInformationLabel>
            <Text>{substanceDescription}</Text>
        </DescriptionContainer>
    );
};

export default SubstanceDescription;
