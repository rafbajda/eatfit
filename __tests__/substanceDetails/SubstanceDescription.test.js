import 'react-native';
import React from 'react';
import renderer from 'react-test-renderer';
import NavigationTestUtils from 'react-navigation/NavigationTestUtils';
import SubstanceDescription from "../../app/screens/substanceDetails/components/SubstanceDescription";

describe('SubstanceDescription snapshot', () => {
    jest.useFakeTimers();
    beforeEach(() => {
        NavigationTestUtils.resetInternalState();
    });

    it('renders component', async () => {
        const tree = renderer.create(<SubstanceDescription />).toJSON();
        expect(tree).toMatchSnapshot();
    });
});
