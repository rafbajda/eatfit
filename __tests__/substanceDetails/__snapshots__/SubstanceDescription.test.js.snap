// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SubstanceDescription snapshot renders component 1`] = `
<RCTScrollView
  automaticallyAdjustContentInsets={false}
  className="sc-bwzfXH rDyNT"
  contentContainerStyle={
    Array [
      Object {
        "padding": undefined,
      },
      undefined,
    ]
  }
  contentInset={
    Object {
      "bottom": 0,
    }
  }
  enableAutomaticScroll={true}
  enableOnAndroid={false}
  enableResetScrollToCoords={true}
  extraHeight={75}
  extraScrollHeight={0}
  getScrollResponder={[Function]}
  handleOnScroll={[Function]}
  keyboardDismissMode="interactive"
  keyboardOpeningTime={250}
  keyboardShouldPersistTaps="handled"
  keyboardSpace={0}
  onScroll={[Function]}
  resetKeyboardSpace={[Function]}
  resetScrollToCoords={
    Object {
      "x": 0,
      "y": 0,
    }
  }
  scrollEventThrottle={1}
  scrollForExtraHeightOnAndroid={[Function]}
  scrollIntoView={[Function]}
  scrollToEnd={[Function]}
  scrollToFocusedInput={[Function]}
  scrollToPosition={[Function]}
  showsVerticalScrollIndicator={true}
  style={
    Object {
      "backgroundColor": "transparent",
      "flex": 1,
    }
  }
  update={[Function]}
  viewIsInsideTabBar={false}
>
  <View>
    <Text
      className="sc-bxivhb czhwhJ"
      style={
        Object {
          "color": "#000",
          "fontFamily": "System",
          "fontSize": 16,
        }
      }
      uppercase={false}
    >
      [missing "en.labels.description" translation]
    </Text>
    <Text
      style={
        Object {
          "color": "#000",
          "fontFamily": "System",
          "fontSize": 16,
        }
      }
      uppercase={false}
    />
  </View>
</RCTScrollView>
`;
